### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
app package version 2

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Review by Salamandar — APPROVED** [review]

**Comment by ericgaspar** [comment]
I have an issue with admin connection... (trying to connect with `admin` and the set password without success)

**Comment by Salamandar** [comment]
@fallen is in the process of upgrading the Django dependency, should be fixed then.

**Comment by Salamandar** [comment]
It *should* be fixed (maybe) now.

**Comment by fallen** [comment]
> It _should_ be fixed (maybe) now.

Hello, 
I have done some testing in a branch for Django 4.2.2 upgrade (latest LTS recently released) but I have not yet taken the time to properly merge it.
I should allocate some time to do this, but the nice weather of this week-end is not helping :)

**Comment by Salamandar** [comment]
Well i have merged a potential workaround for now… should be tested :D

**Comment by ericgaspar** [comment]
!testme

File: doc/POST_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Your credentials for the admin panel are:
+
+- admin username: admin
+- admin password: __PASSWORD__
```
Comment (alexAubin) [inline doc/POST_INSTALL.md]:
```suggestion
- admin username: `admin`
- admin password: the password you chose before install
```

**Review by alexAubin — APPROVED** [review]

**Pull request merged**